team-b-batch/src/main/kotlin/com/address/dsmproject/scheduler/SaveAddressScheduler.kt

```diff
@@ -0,0 +1,45 @@
+package com.address.dsmproject.scheduler
+
+import com.address.dsmproject.scheduler.AddressUtil.ENG_ADDRESS_FILE_PATH
+import com.address.dsmproject.scheduler.AddressUtil.ENG_ADDRESS_ZIP_FILE_PATH
+import com.address.dsmproject.scheduler.AddressUtil.ENG_REQ_TYPE
+import com.address.dsmproject.scheduler.AddressUtil.KOR_ADDRESS_FILE_PATH
+import com.address.dsmproject.scheduler.AddressUtil.KOR_ADDRESS_ZIP_FILE_PATH
+import com.address.dsmproject.scheduler.AddressUtil.KOR_REQ_TYPE
+import com.address.dsmproject.service.SaveAddressService
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+
+@Component
+class SaveAddressScheduler(
+    private val saveAddressService: SaveAddressService,
+) {
+
+    @Scheduled(cron = "0 0 0 1 * * ") // 매달 1일 0시 실행
+    fun saveKoreaAddressScheduler() {
+        deleteIfExistsFile(KOR_ADDRESS_ZIP_FILE_PATH, KOR_ADDRESS_FILE_PATH)
+        saveAddressService.execute(KOR_REQ_TYPE, KOR_ADDRESS_ZIP_FILE_PATH, KOR_ADDRESS_FILE_PATH)
+    }
+
+    @Scheduled(cron = "0 0 0 1 * * ") // 매달 1일 0시 실행
+    fun saveEnglishAddressScheduler() {
+        deleteIfExistsFile(ENG_ADDRESS_ZIP_FILE_PATH, ENG_ADDRESS_FILE_PATH)
+        saveAddressService.execute(ENG_REQ_TYPE, ENG_ADDRESS_ZIP_FILE_PATH, ENG_ADDRESS_FILE_PATH)
+    }
+
+    private fun deleteIfExistsFile(zipFilePath: String, filePath: String) {
```

Comment:
deleteIfExistsFile 은 순수하게 file path 와 같은 identifier 를 받아 제거만 하는 역할을 가져야 코드를 처음 보는 사람이 함수 명을 보고 내부 동작을 추측할 수 있을거 같아요
이와같이 하나의 파일을 받고 하나의 파일을 지우는 역할의 함수로 리팩토링을 하면 좋을 것 같습니다.
```suggestion
    private fun deleteIfExistsFile(filePath: String) {
```

Reply:
혹시 그러면 하나의 파일만을 지우는 함수로 변경한 후 여러번 호출하는게 더 나은 방향일까요?